### PR TITLE
Script to fetch all the wifi passwords 

### LINKED ISSUE #345: Fetching Wifi password and generating a QR Code
### Is your feature request related to a problem? Please describe
Fetching wifi password to easily connect to other devices.

### Describe the solution you'd like
Quickly fetch your WiFi password and generate a QR code for WiFi to allow phones to easily connect.

### PR DESCRIPTION
**Pull request by rahulkarda**
## Description
This is a simple script that fetches all the saved wifi passwords and prints them into the terminal.

Fixes #345 

## Future Improvements

Generating a QR code to easily connect mobile devices.

## Developer's checklist

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-check on my work

**If changes are made in the code:**

- [x] I have followed the [coding guidelines](https://google.github.io/styleguide/jsguide.html)
- [x] My changes in code generate no new warnings
- [ ] My changes are breaking another fix/feature of the project
- [ ] I have added test cases to show that my feature works
- [ ] I have added relevant screenshots in my PR

### CODE REVIEW

File: scripts/wifi_password_checker/script.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+pswd_txt.write(f'{"{:<30}|  {:<}".format("WIFI", "Password")} \n')
+pswd_txt.write(f'{"-" * 50} \n')
+
+print("{:<30}|  {:<}".format('WIFI', 'Password'))
+print("-" * 50)
+
+for i in profiles:
+    try:
+
+        results = subprocess.check_output(['netsh', 'wlan', 'show', 'profile', i, 'key=clear']).decode('utf-8',
```
Comment (bartick) [inline scripts/wifi_password_checker/script.py:20]: None of the profile exists. Unable to find any profile on my device. 
I am using a m1 macbook air

File: scripts/wifi_password_checker/script.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+    except subprocess.CalledProcessError:
+
+        pswd_txt.write(f'{"{:<30}|  {:<}".format(i, "ENCODING ERROR")} \n')
+        pswd_txt.write(f'{"-" * 50} \n')
+        print("{:<30}|  {:<}".format(i, "ENCODING ERROR"))
+        print("-" * 50)
+
+pswd_txt.close()
```
Comment (bartick) [inline scripts/wifi_password_checker/script.py:49]: Didn't quite get how it is generating the QR

**Review by bartick — CHANGES_REQUESTED** [review]

**Comment by rahulkarda** [comment]
Yeah I have to add the QR feature. For now it only shows the saved wifi
passwords in the terminal.

File: scripts/wifi_password_checker/script.py (hunk 1 above)
Comment (rahulkarda) [reply scripts/wifi_password_checker/script.py:20]: @bartick you tried the command "netsh wlan show profile"?

**Review by bartick — APPROVED** [review]

**Review by riyaroy2086 — APPROVED** [review]

**Pull request merged**